lighthouse-core/test/gather/driver-test.js

```diff
@@ -30,37 +27,30 @@ function createOnceStub(events) {
   };
 }
 
-function createSWRegistration(id, url, isDeleted) {
-  return {
-    isDeleted: !!isDeleted,
-    registrationId: id,
-    scopeURL: url,
-  };
+function findSendCommandInvocation(command) {
+  expect(connectionStub.sendCommand).toHaveBeenCalledWith(command, expect.anything());
```

Comment:
it's a little weird to me that it's using the assumed global `connectionStub` here (while `createSendCommandMockFn` is all encapsulated), could we put this as another method on the return value of `createSendCommandMockFn()`? Or have the particular `sendCommand` method passed in

Reply:
yeah sounds good to add to the mock function! 👍 done